### DISCUSSION TITLE
Can I log in to TDLight app by QR code?

### DISCUSSION THREAD

**Discussion by yvasyliev** [tdlight-team/tdlight-java#191] [Q&A] [answered]
I can't find [auth.exportLoginToken](https://core.telegram.org/method/auth.exportLoginToken) implementation in [TDLight docs](https://tdlight-team.github.io/tdlight-docs/tdlight.api/it/tdlight/jni/package-summary.html).
Is it possible to get `login token`?

**Comment by cavallium** [comment, marked as answer]
these apis are not related to tdlib: https://core.telegram.org/method/auth.exportLoginToken
They are related to the mtproto protocol.
If you want authenticate with qr code manually you should use this: https://tdlight-team.github.io/tdlight-docs/tdlight.api/it/tdlight/jni/TdApi.RequestQrCodeAuthentication.html
Otherwise, if you are using SimpleClient, you should use its qr code authentication mode
